tests/kafkatest/services/performance/end_to_end_latency.py

```diff
@@ -36,28 +38,65 @@ def __init__(self, context, num_nodes, kafka, topic, num_records, consumer_fetch
             'num_records': num_records,
             'consumer_fetch_max_wait': consumer_fetch_max_wait,
             'acks': acks,
-            'kafka_opts': self.security_config.kafka_opts
+            'kafka_opts': self.security_config.kafka_opts,
```

Comment:
It may be worth validating that security protocol is plaintext if version is less than 0.9.0.0
Same comment for other services where it isn't already enforced